### DISCUSSION TITLE
How to create an RVE with a certain thickness of grain boundary

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/neper#858] [Q&A] [answered]
Firstly, thank you for your work, it helped a lot in creating polycrystalline models. However, I need to create geometrical models that consider the Set of grain boundaries to achieve polycrystalline organization models with more features and functionality. I want to know how Neper can achieve this feature or if you are considering developing this function in the future.
![002a3746067e452390e56fe30fadeafa](https://github.com/neperfepx/neper/assets/90365358/ef44efd0-cc43-4b3a-b63f-7ef0d50845c6)

**Comment by rquey** [comment, marked as answer]
Not available now, and not very high in the TODO list. Generating zero-thickness boundaries are possible, however.
